### DISCUSSION TITLE
Image automation support for mutable tags

### DISCUSSION THREAD

**Discussion by sachinshakya507** [fluxcd/flux2#4106] [Q&A] [answered]
Flux does not natively support automatically detecting changes when the same tag in a container registry receives a new image (with a different digest) and triggering updates accordingly.

**Comment by hiddeco** [comment, marked as answer]
This is being worked on, see: https://github.com/fluxcd/image-automation-controller/issues/165#issuecomment-1648191191

**Comment by sachinshakya507** [comment]
@hiddeco Thanks for the response
